src/commands/game_commands/play.ts

```diff
@@ -1,15 +1,23 @@
 import GameSession from "../../models/game_session";
+import state from "../../kmq";
 import {
-    sendErrorMessage, getDebugContext, sendInfoMessage, getVoiceChannel, voicePermissionsCheck,
+    sendErrorMessage, getDebugContext, sendInfoMessage, getVoiceChannel, voicePermissionsCheck, getUserIdentifier,
 } from "../../helpers/discord_utils";
 import { getGuildPreference } from "../../helpers/game_utils";
+import { bold } from "../../helpers/utils";
 import BaseCommand, { CommandArgs } from "../base_command";
 import _logger from "../../logger";
 
 const logger = _logger("play");
 
+export enum GameType {
+    ELIMINATION = "elimination",
```

Comment:
Add GameType for default

Reply:
In the validations it'll show that the user can enter either elimination or default if I make this change. Should I allow them to do so?